packages/amplify_datastore/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
+
+* TODO: Describe initial release.
```

Comment:
Now might be a good time to fix this.

Reply:
I'll put tomorrow's date for now hoping to do a release tomorrow.